client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditTable.tsx

```diff
@@ -177,6 +215,232 @@ export const OutboundLineEditTable = ({
     setVvmStatus,
   });
 
+  const packSize =
+    allocateIn.type === AllocateInType.Packs ? allocateIn.packSize : undefined;
+
+  const unit = Formatter.sentenceCase(item?.unitName ?? t('label.unit'));
+  const pluralisedUnitName = getPlural(unit, 2);
+
+  const mrtColumns = useMemo(() => {
+    const cols: ColumnDef<DraftStockOutLineFragment>[] = [
+      {
+        id: 'canAllocate',
+        header: '',
+        accessorFn: row => canAutoAllocate(row, packSize),
+        size: 0,
+        // TO-DO: Create "CheckboxCell" component
+        Cell: ({ renderedCellValue }) => {
+          return renderedCellValue ? '✓' : '';
+        },
+        defaultHideOnMobile: true,
+      },
+      {
+        accessorKey: 'batch',
+        header: t('label.batch'),
+        size: 60,
+        Cell: ({ row }) => {
+          return (
+            <div style={{ display: 'flex', alignItems: 'center' }}>
+              {row.original.batch}
+              {row.original.itemVariantId && (
+                <ItemVariantInfoIcon
+                  includeDoseColumns={allocateIn.type === AllocateInType.Doses}
+                  itemId={item?.id ?? ''}
+                  itemVariantId={row.original.itemVariantId}
+                />
+              )}
+            </div>
+          );
+        },
+      },
+      {
+        accessorKey: 'expiryDate',
+        header: t('label.expiry-date'),
+        size: 100,
+        columnType: ColumnType.Date,
+        Cell: ({ cell }) => {
+          return <ExpiryDateCell cell={cell} />;
+        },
+      },
+      {
+        accessorKey: 'vvmStatus',
+        header: t('label.vvm-status'),
+        Cell: ({ row }) => (
+          <VVMStatusSearchInput
+            onChange={(vvmStatus: VvmStatusFragment | null | undefined) => {
```

Comment:
were you planning to fix these, so the inputs disable when its a non-allocatable line?

Reply:
You shouldn't be able to set VVM or packs issued for non-allocatable lines. But you can change between valid VVM statuses for there (i.e. stage 1 to stage 2, as a spot check before creating the shipment)